README.md

```diff
@@ -1,24 +1,53 @@
-# README
+# GRAB BAG
+## The hottest new way to store and share your files
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+View it live at: https://grab-bag-1701.herokuapp.com/
 
-Things you may want to cover:
+### Setup
 
-* Ruby version
+To set up a local copy of this project, perform the following:
 
-* System dependencies
+* Clone the repository: `git clone https://github.com/glassjoseph/grab_bag.git`
+* `cd` into the project's directory
+* Run `bundle install`
+* Run `bundle exec rake db:{create,migrate,seed}` to set up the database locally and seed it with files, folders, users, and comments.
+* To login as an admin (locally), use these credentials - email: admin@admin.com, password: password
```

Comment:
Have we set this up yet? Do we have admin user stories? Feels like the last remaining thing. Should 'locally' be 'in development'?